CyberPhish.py

```diff
@@ -3,7 +3,7 @@
 from sys import version_info
 
 if version_info<(3,0,0):
-    print('[!] Please use Python 3. $ python3 CyberPhish.py')
+    print('[!] Please use Python 3 for windows . $ python3 CyberPhish.py in linux')
```

Comment:
Since these are shell commands, for Windows it'd be plain `python`. For macOS/Darwin and Linux it'd be `python3`